css/src/components/toggle.css

```diff
@@ -0,0 +1,76 @@
+.yoast-feature {
+	max-width: 600px;
+}
+
+.yoast-toggle__item {
+	display: flex;
+	justify-content: space-between;
+	padding-bottom: 16px;
+	border-bottom: 1px solid var(--color-border);
+	margin-bottom: 16px;
+}
+
+.yoast-toggle__item-title {
+	display: flex;
+	font-weight: bold;
+	align-items: center;
+}
+
+.yoast-toggle input[type="checkbox"], .yoast-toggle-inverse input[type="checkbox"] {
+	display: none;
```

Comment:
We can't use display none to hide the checkboxes.
By doing so, they're hidden visually _and_ from assistive technologies. Nothing is announced.
They're not even focusable with a keyboard.

Reply:
Looking at the previous implementation, the checkbox was hidden with `opacity: 0;` and then positioned absolutely to remove it from the layout. I'd suggest to give it a try.

Reply:
I made a class `visually-hidden` with this CSS:
```
	clip: rect(0 0 0 0);
	clip-path: inset(50%);
	height: 1px;
	overflow: hidden;
	position: absolute;
	white-space: nowrap;
	width: 1px;
```

I now have focus. Added a focus style and I can now toggle the switch my keyboard. 
Got the CSS example from: https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html